build.sbt

```diff
@@ -208,7 +208,12 @@ lazy val testRunner = crossProject(JVMPlatform, JSPlatform)
     ),
     mainClass in (Test, run) := Some("zio.test.sbt.TestMain")
   )
-  .jsSettings(libraryDependencies ++= Seq("org.scala-js" %% "scalajs-test-interface" % "0.6.29"))
+  .jsSettings(
+    libraryDependencies ++= Seq(
+      "org.scala-js"      %% "scalajs-test-interface" % "0.6.29",
+      "io.github.cquiroz" %%% "scala-java-time"       % "2.0.0-RC3" % Test
```

Comment:
I think we can revert this since we're not actually running any tests on ScalaJS.

Reply:
Sorry, what command are you running that would you give you an error?

Reply:
`test` inside sbt. We don't run this on CI, because we run `testJVM` and `testJS`. But it should be usual for people to run `test` before pushing.

Reply:
Ah, I see. Can we add `scala-java-time` as a test dependency in the root project?

Reply:
I think it would be hard, because it's valid only for JS but `root` aggregates JVM, JS, documentation and benchmarks.

Reply:
How about?

```
.jsSettings(libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.0.0-RC3" % Test)
```

Alternatively we could do this.

```scala
addCommandAlias("test", "testJVM;testJS")
```

Reply:
I cannot add it `root`, it's `Project` it doesn't have `.jsSettings`.
There is `.jsSettings(libraryDependencies += "io.github.cquiroz" %%% "scala-java-time" % "2.0.0-RC3" % Test)` at each JS project that has tests requiring scala-java-time.
Making `test` a command alias is somehow dangerous, because if someone forgets to add new project `testJVM` and `testJS`, then tests won't be run. OTOH some projects require `test:run` instead of `test` and also recursing with `test` to projects without tests is redundant.
So I'm reverting addition of dependencies to projects that tests are not supposed to be run by invoking `testJS` and adding command alias.